src/ontology/edits/oeo-shared.omn

```diff
@@ -1433,17 +1433,17 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1879"
 Class: OEO_00000315
 
     Annotations: 
-        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-shared",
-        <http://purl.obolibrary.org/obo/IAO_0000115> "An organisation role is a role of an aggregate of people that has a collective goal and a set of organisation rules.",
-        <http://purl.obolibrary.org/obo/IAO_0000118> "organization role",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/314
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/314
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/370
 rework module structure 
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/1592
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652
 remove disjointness with person
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/1702
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1716",
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-shared",
```

Comment:
```suggestion
```
Remove artifact. The "belongs to module" annotation is deprecated.

Reply:
I agree